tests/network-tests/src/scenarios/carthageNPoSSwitch.ts

```diff
@@ -0,0 +1,12 @@
+import changingAuthorities from 'src/flows/carthage/changingAuthorities'
+import claimingPayoutsEnabled from 'src/flows/carthage/claimingPayoutEnabled'
+import setForceNewEra from 'src/flows/carthage/setForceNewEra'
+import currentEraIsSomeInPoS from 'src/flows/carthage/currentEraIsSomeInPoS'
+import { scenario } from '../Scenario'
+
+scenario('Carthage NPoS switch', async ({ job }) => {
+  job('set force era to force new', setForceNewEra)
+  job('current era is none', currentEraIsSomeInPoS)
```

Comment:
jobs in general will run in parallel. In this scenario it makes more sense for job running setForNewEra flow to be the dependency for the remaining jobs. This can be done with the `after` or `require` methods like so:

```
const forceNewEraJob = job('set force era to force new', setForceNewEra)
// will run after forceNewEraJob only if it is successful.
job('current era is none', currentEraIsSomeInPoS).requires(forceNewEraJob)
.
.
```

Reply:
I put all the PoA / NPoS assertions in one job